docs/api-ref.md

```diff
@@ -576,9 +574,10 @@ REST API: [Publish Datasource](https://help.tableau.com/current/api/rest_api/en-
 Name | Description
 :--- | :---
 `datasource_item`  |  The `datasource_item` specifies the new data source you are adding, or the data source you are appending to. If you are adding a new data source, you need to create a new `datasource_item` with a `project_id` of an existing project. The name of the data source will be the name of the file, unless you also specify a name for the new data source when you create the instance. See [DatasourceItem](#datasourceitem-class).
-`file_path`  |  The path and name of the data source to publish.
+`file`  |  The file path or file object of the data source to publish.
```

Comment:
Could you make these same changes (rename file_path->file and add as_job) to the call above so they match:
`datasources.publish(datasource_item, file_path, mode, connection_credentials=None)`

Reply:
Done